code/modules/gear_presets/upp.dm

```diff
@@ -3786,6 +3786,8 @@
 
 //*****************************************************************************************************/
 
+/datum/equipment_preset/upp/doctor
+	name = "UPP Doktor"
```

Comment:
why's this here instead of editing the other definition

Reply:
I think it might've been fucked with bc of cuberounds UPP PRs, should I change it?

Reply:
> I think it might've been fucked with bc of cuberounds UPP PRs, should I change it?

just change the name of the new /datum/equipment_preset/upp/doctor since you merged master

Reply:
yeah will do